tests/test_single_m4b_merge.py

```
@@ -48,7 +48,7 @@ def test_download_cover(self):
         m4b = self.m4b_data(primary_asin)
         m4b.download_cover()
         assert ((test_cover).exists() and
-                os.path.getsize(test_cover) == 779312)
+                os.path.getsize(test_cover) == 779388)
```

Ruff (0.15.15)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)